docs/javascript/quickstart.md

```diff
@@ -0,0 +1,685 @@
+---
+title: Ionic JavaScript Quickstart
+sidebar_label: Quickstart
+---
+
+<head>
+  <title>Ionic JavaScript Quickstart: JavaScript Basics</title>
+  <meta
+    name="description"
+    content="Ionic JavaScript Quickstart covers the basics of JavaScript and Ionic, including any Ionic-specific features. Learn how to build JavaScript apps."
+  />
+</head>
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+
+Welcome! This guide will walk you through the basics of Ionic JavaScript development using Vite. You'll learn how to set up your development environment, create a simple project, explore the project structure, and understand how Ionic components work. This is perfect for getting familiar with Ionic JavaScript before building your first real app.
+
+If you're looking for a high-level overview of what Ionic JavaScript is and how it fits into the web development ecosystem, see the [Ionic JavaScript Overview](overview).
+
+## Prerequisites
+
+Before you begin, make sure you have Node.js and npm installed on your machine.
+You can check by running:
+
+```shell
+node -v
+npm -v
+```
+
+If you don't have Node.js and npm, [download Node.js here](https://nodejs.org/en/download) (which includes npm).
+
+## Create a Project with Vite
+
+First, create a new Vite project with vanilla JavaScript:
+
+```shell
+npm create vite@latest my-app -- --template vanilla
```

Comment:
The app is named `my-app` here instead of `myApp` like the other frameworks because it will prompt for the name again if you don't use kebab case.